lib/mayaUsd/ufe/wrapUtils.cpp

```diff
@@ -236,6 +249,7 @@ void wrapUtils()
     // representation of Ufe::Path as comma-separated segments.  We know that
     // the USD path separator is '/'.  PPT, 8-Dec-2019.
     def("getStage", getStage);
+    def("getAllStages", getAllStages, return_value_policy<PXR_NS::TfPySequenceToList>());
```

Comment:
Return value policy from example Pixar USD code, e.g.
https://github.com/PixarAnimationStudios/USD/blob/dc710925675f7f58f7a37f6c18d046a687b09271/pxr/usd/usd/wrapStageCache.cpp#L63
otherwise Boost Python complains at run-time that there is no converter for a vector of stages.